src/main/java/com/epam/training/model/ChristmasGifts.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.training.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ChristmasGifts {
```

Comment:
just Gift

Reply:
@FilippStankevichTR Done.
Please, see the changes here: https://github.com/AIniuseva/epam-oop-task/pull/1/files#diff-f15b04e725e976bbe8fdb043f8dbf6eb9c8e53318e1c6e40ef65c951db26f41eR6